cjt/jenkins.yaml

```diff
@@ -11,7 +35,7 @@ unclassified:
               git:
                 remote: "https://github.com/cloudy-demos/pipeline-libraries.git"
 jobs:
-  - >
+  - script : >
```

Comment:
what is this for?

Reply:
the newer configuration-as-code version has slightly different syntax for job configuration